### DISCUSSION TITLE
Convert Collection into array

### DISCUSSION THREAD

**Discussion by stefaanMLB** [mikro-orm/mikro-orm#2884] [General]
Is it possible to convert a collection into an array so that the typical array function slike .map() .filter() .find() etc can be used ?

**Comment by stefaanMLB** [comment]
.toArray() works but creates copies of the entities. 
Wgy does it create copies ?  Can be convenient to update one or more entities returned from .toArray()

**Reply by B4nan** [reply]
They are not copies, they are DTOs. To get the entities, just do `coll.getItems()`.
